rustfmt-core/rustfmt-lib/src/config/config_type.rs

```diff
@@ -50,6 +50,22 @@ impl ConfigType for IgnoreList {
     }
 }
 
+macro_rules! update_config {
```

Comment:
Do we still need this macro?

Reply:
Yep, see the usage in `fill_from_parsed_config` on line 164.